docs/gpa.md

```diff
@@ -233,78 +248,100 @@ The information included in the notice of intended procurement covers all the in
     <tbody>
       <tr class="section">
         <td>X:7</td>
-        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
-      </tr>
+        <td colspan=2>A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
+     </tr>
```

Comment:
Not sure why quoting and whitespace were changed.

```suggestion
        <td colspan="2">A procuring entity shall make available to suppliers tender documentation that includes all information necessary to permit suppliers to prepare and submit responsive tenders. Unless already provided in the notice of intended procurement, such documentation shall include a complete description of:</td>
      </tr>
```

Reply:
Done